### PR TITLE
feat: Documentation detailing new telemtry that is being introduced into Pact

### PR DESCRIPTION
**Pull request by Inksprout**
The purpose of this PR is to outline the purpose, intention and specifics of the telemetry that will be rolled out as part of ongoing development of Pact. This document is the outcome of discussions with Pact maintainers and describes the type of data that will be collected, as well as how to opt-out of sending telemetry.

### CODE REVIEW

File: website/docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: Telemetry
+---
+
+At Pact, we strive to develop great open source tools for the community. We rely on the community's input to help us iterate on and improve Pact. Telemetry is additional information that helps us to better understand the community's needs, diagnose issues, and prioritise feature work.
```
Comment (uglyog) [inline website/docs/telemetry.md:5]: Pact is not a place

File: website/docs/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+```
+Please note: we are tracking events anonymously to gather important usage statistics like Pact-Ruby
+version and operating system. To disable tracking, set the 'PACT_DO_NOT_TRACK' environment variable
+to 'true'."
+```
+
+To opt out a 'PACT_DO_NOT_TRACK' environment variable needs to be set to 'true' in the environment you are using Pact in.
+
+```
+export PACT_DO_NOT_TRACK='true'
```
Comment (uglyog) [inline website/docs/telemetry.md:28]: Example is a bit vague

File: website/docs/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+| Data                        | Description                                                                                                  |
+| :-------------------------- | :----------------------------------------------------------------------------------------------------------- |
+| Version of API              | Version of the GA API used to collect telemetry                                                              |
+| Hash of hostname            | MD5 digest hash of the hostname. Used to anonymously detect repeat usage                                     |
+| Human readable library name | Human readable string to indicate which library generated the event e.g 'Pact JS'                            |
+| Library name                | The name of the library as it appears in github e.g 'pact-js'                                                |
+| Library version             | The semantic version of the Library that generated the event e.g v1.2.3                                      |
+| AIP                         | Anonymise IP flag set to '1' so that the IP address of the user will be anonymised when collecting telemetry |
```
Comment (uglyog) [inline website/docs/telemetry.md:44]: change to: IP address of the computer

**Review by uglyog — APPROVED** [review]

File: website/docs/telemetry.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+When using Pact locally a message will display in the terminal indicating that data will be collected. This message will contain instructions to disable telemetry if desired.
+
+```
+Please note: we are tracking events anonymously to gather important usage statistics like Pact-Ruby
+version and operating system. To disable tracking, set the 'PACT_DO_NOT_TRACK' environment variable
+to 'true'."
+```
+
+To opt out a 'PACT_DO_NOT_TRACK' environment variable needs to be set to 'true' in the environment you are using Pact in.
```
Comment (mefellows) [inline website/docs/telemetry.md:25]: > In macOS and Linux operating systems, you can disable telemetry for a single session. To disable telemetry for your current session, run the following command to set the environment variable PACT_DO_NOT_TRACK to "true". You must repeat the command for each new terminal or session.

```
export PACT_DO_NOT_TRACK='true'
```

> To disable telemetry in Windows:

```
setx PACT_DO_NOT_TRACK "true"
refresh env
```

File: website/docs/telemetry.md (hunk 2 above)
Comment (mefellows) [reply website/docs/telemetry.md:28]: How about:

> In macOS and Linux operating systems, you can disable telemetry for a single session. To disable telemetry for your current session, run the following command to set the environment variable PACT_DO_NOT_TRACK to "true". You must repeat the command for each new terminal or session.

```
export PACT_DO_NOT_TRACK='true'
```

> To disable telemetry in Windows:

```
setx PACT_DO_NOT_TRACK "true"
refresh env
```

File: website/docs/telemetry.md (hunk 3 above)
Comment (mefellows) [inline website/docs/telemetry.md:44]:
```suggestion
| AIP                         | Anonymised IP address of the computer |
```

Comment (mefellows) [reply website/docs/telemetry.md:44]:
```suggestion
| AIP                         | Anonymised IP address of the computer |
```

File: website/docs/telemetry.md (hunk 1 above)
Comment (mefellows) [reply website/docs/telemetry.md:5]: How can we word this better to refer to us as a collective? How about:

```suggestion
The Pact team strive to develop great open source tools for the community. We rely on the community's input to help us iterate on and improve Pact. Telemetry is additional information that helps us to better understand the community's needs, diagnose issues, and prioritise feature work.
```

**Review by mefellows — APPROVED** [review]
I've added some minor changes. Looks great, thank you.

File: website/sidebars.json [hunk 5]
```diff
@@ -145,7 +145,8 @@
       ]
     },
     "implementation_guides/workshops",
-    "faq"
+    "faq",
+    "telemetry"
```
Comment (mefellows) [inline website/sidebars.json:149]: Last one, I think this is best put as the last item in the "docs" section.

Comment (Inksprout) [reply website/sidebars.json:149]: you mean on the "Pact Docs" sidebar don't you?

**Comment by mefellows** [comment]
Yes please, I think the section in the sidebar JSON is called "docs"

**Pull request merged**